### ISSUE TITLE
Conflict with Bulma css framework

### ISSUE THREAD

**Issue by li-rongcheng** [ckeditor/ckeditor5#876]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

"@ckeditor/ckeditor5-build-classic": "^1.0.0-alpha.2"

## 📋 Steps to reproduce

1. use bulma:
`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.2/css/bulma.min.css">`

2. refresh page that has ckeditor5 embedded

3. Apply headings like "Heading 1", "Heading 2"

## ✅ Expected result

The font size should be changed to the relevant heading font.

## ❎ Actual result

The font size did not changed, e.g. the correct result should be:
![image](https://user-images.githubusercontent.com/4549187/37073243-c5584e50-2219-11e8-8864-28ebbd73c8e2.png)
the actual result was:
![image](https://user-images.githubusercontent.com/4549187/37073263-e0084822-2219-11e8-95ed-da9cf0b99487.png)

## 📃 Other details that might be useful

The bulleted list and number list don't work correctly either.

I have uploaded the source to https://github.com/li-rongcheng/electron-angular5-ckeditor5 .
To reproduce the issue, just reapply the bulma css link in src/index.html then do: ng serve

I have confirmed this issue was caused by bulma. I report the issue to ckeditor5 is because I think ckeditor5 should work with any css framework.

**Comment by Reinmar** [comment]
The content you edit is part of the website in which the editor is included. So you need to provide a proper styles for the editor content or avoid overriding the native styles. If you use a CSS framework which resets CSS instead of normalizing them, then you'll need to re-style the content.

After the beta is released such styles could look like this:

```css
.ck-content h1 {
    font-size: 2em; 
}
.ck-content h2 {
    font-size: 1.8em; 
}
```

Prior to beta (so right now) you'd need to use a different selector. E.g. the editor's editable:

```css
.ck-editor__editable h1 {
    font-size: 2em; 
}
ck-editor__editable h2 {
    font-size: 1.8em; 
}
```

But, in both cases, you'll need to remember about ensuring proper specificity of these selectors. So e.g. if Bulma also uses a single class + element, then you'd need to put CKEditor content styles lower. If it uses e.g. two classes + element, then you'd need to use at least two classes + element as well. 

In other words, general CSS rules apply to the editor contents.

**Comment by Reinmar** [comment]
PS. We're planning to create a guide in which we'll share example content styles. But that's future.

**Comment by li-rongcheng** [comment]
Do you mean I need to provide an additional css definition if I use ckeditor5 with bulma?

But when I use ckeditor4 with bulma, there's no any problem, so I still think that might be a ckeditor5's bug. 

If it's not a bug, the document may need to remind users that when use with some css frameworks (I also notice some style changes with bootstrap4 but that's much better than bulma), an additional css definition must be provide and how it should be done correctly.

**Comment by li-rongcheng** [comment]
@Reinmar Sorry I didn't @ you for my previous message.
